tests/integration/components/file-dropzone-test.js

```diff
@@ -0,0 +1,125 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { dragAndDrop, dragEnter, dragLeave } from 'ember-file-upload/test-support';
+
+module('Integration | Component | file-dropzone', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('dropping a file calls ondrop', async function(assert) {
+    this.onDrop = (dataTransfer) => {
+      dataTransfer.files.forEach((file) => assert.step(file.name));
+    };
+    /**
+     Must set `allowUploadsFromWebsites=true` otherwise `FileDropzone.isAllowed` returns false
+    */
```

Comment:
This kind of sucks, an alternative would be to detect if the component is being used in a test environment within `isAllowed`.

Reply:
This is possible no? Importing ENV?

Reply:
Resolved in e7c85a7.